core/src/main/java/org/mskcc/cbio/oncokb/util/AlterationUtils.java

```diff
@@ -338,9 +338,19 @@ private static String findHighestOncogenic(List<Evidence> evidences) {
     public static List<Alteration> excludeVUS(List<Alteration> alterations) {
         List<Alteration> result = new ArrayList<>();
         Set<Alteration> VUS = new HashSet<>();
-        Set<Gene> allGenes = CacheUtils.getAllGenes();
+        Set<Gene> allGenes = new HashSet<>();
+        if(CacheUtils.isEnabled()) {
+            allGenes = CacheUtils.getAllGenes();
+        }else {
+            allGenes = new HashSet<>(ApplicationContextSingleton.getGeneBo().findAll());
```

Comment:
Nope, this statement will only be used if the cache is not available. In that case, we don't need to set back to cache.